### PR TITLE
chatterino2: update to 2.3.4.

### PR DESCRIPTION
**Pull request by tibequadorian**
supersedes #32199

#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

File: srcpkgs/chatterino2/template [hunk 1]
```diff
@@ -1,55 +1,52 @@
...
-	find \( -name '*.cpp' -o -name '*.hpp' \) \
-		-exec sed -i "{}" -e 's/Communi::/::/g' \;
+	rmdir lib/libcommuni lib/serialize lib/settings lib/signals lib/rapidjson \
+	      lib/websocketpp lib/qtkeychain cmake/sanitizers-cmake
+	mv "${XBPS_BUILDDIR}/libcommuni-${_libcommuni_commit}" "lib/libcommuni"
+	mv "${XBPS_BUILDDIR}/serialize-${_serialize_commit}" "lib/serialize"
+	mv "${XBPS_BUILDDIR}/settings-${_settings_commit}" "lib/settings"
+	mv "${XBPS_BUILDDIR}/signals-${_signals_commit}" "lib/signals"
+	mv "${XBPS_BUILDDIR}/qtkeychain-${_qtkeychain_commit}" "lib/qtkeychain"
+	mv "${XBPS_BUILDDIR}/sanitizers-cmake-${_sanitizers_cmake_commit}" "cmake/sanitizers-cmake"
```
Comment (paper42) [inline srcpkgs/chatterino2/template:49]: Can we use create_wrksrc to have this all contained in one directory that will be cleaned with xbps-src clean?

Comment (tibequadorian) [reply srcpkgs/chatterino2/template]: That would require us to modify the patches accordingly to $build_wrksrc on every update :/ I would leave it as it is for now. Or do you know a better way?

Comment (tibequadorian) [reply srcpkgs/chatterino2/template]: I also could set `build-wrksrc=chatterino2` and `mv "$pkgname-$version chatterino2`. Does that sound like a good solution?

**Comment by tibequadorian** [comment]
Thanks for merging @paper42. At the end I wasn't satisfied with either solution so I decided to just leave it as it is. It works for now and can still be improved once #36409 is merged.

**Pull request merged**